modules/end-user-guide/pages/supported-languages.adoc

```diff
@@ -1,10 +1,18 @@
 :_content-type: REFERENCE
-:description: Supported languages
+:description: Stack samples
 :keywords: user-guide, languages
-:navtitle: Supported languages
+:navtitle: Stack samples
 :page-aliases:
 
 [id="supported-languages"]
```

Comment:
I suggest changing the file name from `supported-languages.adoc` to `stack-samples.adoc` and updating `nav.adoc`.
Also the id needs to be updated:
```suggestion
[id="stack-samples"]
```